src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
```

Comment:
Why A and B? You can use prev and current, it's more meaningful

Reply:
I would think of something like `prevItem` and `currentItem` here